src/component/handlers/edit/editOnBeforeInput.js

```diff
@@ -173,11 +173,25 @@ function editOnBeforeInput(editor: DraftEditor, e: SyntheticInputEvent): void {
     editor._waitingOnInput = true;
```

Comment:
Given that this calls out into external code I'm going to leave it the way it is now.